packages/invoices/src/OutboundShipment/Dashboard/Widget.tsx

```diff
@@ -9,13 +9,13 @@ import {
   useNotification,
   useOmSupplyApi,
   useQuery,
+  StatsPanel,
+  Widget,
 } from '@openmsupply-client/common';
-import Widget from './Widget';
-import { StatsPanel } from '../StatsPanel';
 import { useNavigate } from 'react-router';
-import { getOutboundShipmentCountQueryFn } from '../api';
+import { getOutboundShipmentCountQueryFn } from './api';
 
-export const OutboundShipmentsWidget: React.FC = () => {
+export const OutboundShipmentWidget: React.FC = () => {
```

Comment:
Cool! Might actually not be too much work to do the 'mini' dashboards for the top level navigation areas like `/distribution` etc with widgets like these?

Reply:
yeah, should be pretty straightforward.. so maybe that idea is back on the table 😁